syntax = "proto3";

// import "google/protobuf/timestamp.proto";

message LocationIngestMessage {
    int32 person_id = 1;
    string longitude = 2;
    string latitude = 3;
    string creation_time = 4;
}

service LocationIngest {
    rpc Create(LocationIngestMessage) returns (LocationIngestMessage);
}